src/bin/cargo-ziggy/fuzz.rs

```diff
@@ -11,422 +11,496 @@ use std::{
+                    0 => "-l2",
+                    1 => "-l2a",
+                    _ => "-c-", // disable Cmplog, needs AFL++ 4.08a
+                };
+                // AFL timeout is in ms so we convert the value
+                let timeout_option_afl = match self.timeout {
+                    Some(t) => format!("-t{}", t * 1000),
+                    None => String::new(),
+                };
+                let dictionary_option = match &self.dictionary {
+                    Some(d) => format!("-x{}", &d.display().to_string()),
+                    None => String::new(),
+                };
+                let log_destination = || match job_num {
+                    0 => File::create(format!("output/{}/logs/afl.log", self.target))
+                        .unwrap()
+                        .into(),
+                    _ => process::Stdio::null(),
+                };
+
+                fuzzer_handles.push(
+                    process::Command::new(cargo.clone())
+                        .args(
+                            [
+                                "afl",
+                                "fuzz",
+                                &fuzzer_name,
+                                &format!("-i{}", &self.parsed_corpus()),
+                                &format!("-p{power_schedule}"),
+                                &format!("-ooutput/{}/afl", self.target),
+                                &format!("-g{}", self.min_length),
+                                &format!("-G{}", self.max_length),
+                                &use_shared_corpus,
+                                &use_initial_corpus_dir,
+                                &format!(
+                                    "-V{}",
+                                    self.minimization_timeout + SECONDS_TO_WAIT_AFTER_KILL
+                                ),
+                                old_queue_cycling,
+                                cmplog_options,
+                                mopt_mutator,
+                                &timeout_option_afl,
+                                &dictionary_option,
+                                &format!("./target/afl/debug/{}", self.target),
+                            ]
+                            .iter()
+                            .filter(|a| a != &&""),
+                        )
+                        .env("AFL_AUTORESUME", "1")
+                        .env("AFL_TESTCACHE_SIZE", "100")
+                        .env("AFL_FAST_CAL", "1")
+                        .env("AFL_FORCE_UI", "1")
+                        .env("AFL_IGNORE_UNKNOWN_ENVS", "1")
+                        .env("AFL_CMPLOG_ONLY_NEW", "1")
+                        .env("AFL_DISABLE_TRIM", "1")
+                        .env("AFL_NO_WARN_INSTABILITY", "1")
+                        // marc: 1 costs performance, 15 maybe? putting 5 for now.
+                        .env("AFL_FUZZER_STATS_UPDATE_INTERVAL", "5")
```

Comment:
could we crank this up? what is the absolute minimum value that you need?

Reply:
Something that makes the ziggy interface look alive, so I guess 10 seconds would be a good compromise?